examples/mamba/mamba.py

```diff
@@ -98,6 +104,7 @@ def __init__(
         self.dt_rank = math.ceil(self.d_model / 16) if dt_rank == "auto" else dt_rank
         self.use_fast_path = use_fast_path
         self.layer_idx = layer_idx
+        self.is_inference = is_inference
```

Comment:
is_inference can be defined by checking if store has been created and not null. It's better than having a param for this imo

Reply:
or we can remove the is_inference param and just check if store is defined, because we only check for inference if we want to use the store